autoload/nerdtree/ui_glue.vim

```diff
@@ -108,24 +108,24 @@ function! s:customOpenBookmark(node) abort
 endfunction
 
 "FUNCTION: s:initCustomOpenArgs() {{{1
-" Make sure NERDTreeCustomOpenArgs has needed keys
 function! s:initCustomOpenArgs() abort
     let l:defaultOpenArgs = {'file': {'reuse': 'all', 'where': 'p'}, 'dir': {}}
     let l:customOpenArgs = get(g:, 'NERDTreeCustomOpenArgs', {})
-    return extend(l:customOpenArgs, l:defaultOpenArgs, 'keep')
 
-    if v:false is# s:validateType(l:customOpenArgs, v:t_dict)
+    if v:false is# s:validateType(l:customOpenArgs, type({})) || empty(l:customOpenArgs)
```

Comment:
I don't know if anyone still uses it, but NERDTree is supposed to be compatible to 7.3. After reviewing (the 7.3 documentation](http://vimdoc.sourceforge.net/htmldoc/eval.html#expression-syntax), I think `is#` and `v:false` didn't exist at that time. Let's use this instead.
```suggestion
    if !s:validateType(l:customOpenArgs, type({})) || empty(l:customOpenArgs)
```